### DISCUSSION TITLE
taking to long to load

### DISCUSSION THREAD

**Discussion by narijeong** [community/community#15555] [Projects and Issues]
I have been trying to open this page

https://github.com/kubernetes-sigs/aws-ebs-csi-driver

but it keep showing me the page saying "its's taking too long to load"

Can someone help?

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Comment by jfullstackdev** [comment]
that did not take too long to load on my end 

![image](https://user-images.githubusercontent.com/47092464/194710381-aeffaedb-2c0e-4523-9825-6db7f4fa35c3.png)

it will depend on your Internet connection, memory, installed extensions, etc. or your ISP designed to block GitHub or some selected URLs, like in China
